ui/pages/Search/components/filters/LocusListSelector.jsx

```diff
@@ -24,8 +23,34 @@ class BaseLocusListDropdown extends React.Component {
 
   componentDidUpdate(prevProps) {
     const { locusList, onChange } = this.props
+
     if (prevProps.locusList.rawItems !== locusList.rawItems) {
-      const { locusListGuid, rawItems } = locusList
+      const { locusListGuid } = locusList
+      let { rawItems } = locusList
+
+      if (locusList.paLocusList) {
+        const grouped = locusList.items?.reduce((acc, item) => {
+          const confidence = item.pagene?.confidenceLevel || 0
+          const group = acc[confidence] || []
+          group.push(item.display)
+          acc[confidence] = group
+          return acc
+        }, {})
+        if (grouped) {
+          rawItems = {
+            green: grouped['3']?.concat(grouped['4'])?.join(', ') || '',
+            amber: grouped['2']?.join(', ') || '',
+            red: grouped['1']?.join(', ') || '',
+          }
+        } else {
```

Comment:
what would the case be that there is no `grouped` but we would still want to display the 3 panels?

Reply:
None, so the else clause is redundant.